src/main/configurations/Translate/Common/xsl/GetZdsZaakFromZgwZaak.xslt

```diff
@@ -16,6 +16,17 @@
     	</xsl:choose>
 	</xsl:function>
 
+    <xsl:function name="zgw:convertZgwBetalingsIndicatieToZdsBetalingsIndicatie">
+        <xsl:param name="zdsBetalingsIndicatie"/>
+		<xsl:choose>
+            <xsl:when test="lower-case($zdsBetalingsIndicatie)='nvt'">N.v.t.</xsl:when>
+            <xsl:when test="lower-case($zdsBetalingsIndicatie)='nog_niet'">(Nog) niet</xsl:when>
+            <xsl:when test="lower-case($zdsBetalingsIndicatie)='gedeeltelijk'">Gedeeltelijk</xsl:when>
+            <xsl:when test="lower-case($zdsBetalingsIndicatie)='geheel'">Geheel</xsl:when>
+            <xsl:otherwise>N.v.t.</xsl:otherwise>
```

Comment:
Weirdly enough this needs to be empty string. According to ZDS and ZGW schema's the value can be one of the enum values OR empty string. Because of that it has a slightly different meaning than N.v.t. An empty string is like Schrödinger's cat. Until we see a concrete betalingsindicatie as one of the enum options, we dont know if there is a betalingsindicatie or not.